101-integrationpatterns-messagerouter-servicebus/azuredeploy.json

```diff
@@ -0,0 +1,199 @@
+{  
+    "$schema":"https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion":"1.0.0.0",
+    "parameters":{  
+       "MessageRouterServiceBusNamespacePrefix":{  
+          "defaultValue":"MessageRouterServiceBus",
+          "type":"string",
+          "metadata":{  
+             "description":"The prefix of the name for the Service Bus namespace, will be appended with a unique string."
+          }
+       },
+       "IncomingDeliveryRequestsTopicName":{  
+          "defaultValue":"incomingdeliveryrequests",
+          "type":"string",
+          "metadata":{  
+             "description":"The name for the Service Bus Topic."
+          }
+       }
+    },
+    "variables":{  
+        "serviceBusNamespaceName": "[concat(parameters('MessageRouterServiceBusNamespacePrefix'), '-', uniquestring(resourceGroup().id))]" 
+    },
+    "resources":[  
+       {  
+          "type":"Microsoft.ServiceBus/namespaces",
```

Comment:
Can you re-order the properties on each resource to be consistent?  It's hard to digest/review when the properties are in misc locations... e.g.

type
apiVersion
name
location
dependsOn